cluster/gce/gci/mounter/Makefile

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 TAG=v2
-REGISTRY=staging-k8s.gcr.io
```

Comment:
I'm not sure `staging-k8s.gcr.io` applies to this migrations, the subdomain `staging-k8s` is different thatn `k8s`

Reply:
Agreed 👍